src/plugins/auto/plugin.cpp

```diff
@@ -433,16 +434,19 @@ DeviceInformation MultiDeviceInferencePlugin::SelectDevice(const std::vector<Dev
             auto& gpuUniqueName = item.uniqueName;
             if (gpuUniqueName.find("iGPU") != std::string::npos) {
                 iGPU.push_back(item);
-            } else if (gpuUniqueName.find("dGPU") != std::string::npos) {
-                dGPU.push_back(item);
+            } else if (gpuUniqueName.find("0x4f87") != std::string::npos) {
+                dGPU_DG2.push_back(item);
+            } else if (gpuUniqueName.find("Intel(R) Iris(R) Xe MAX Graphics") != std::string::npos) {
```

Comment:
please don't use device id or device names as it's not future proof an usually not portable. GPU_UARCH_VERSION and DEVICE_TYPE metrics can be used instead

Reply:
thanks, @vladimir-paramuzov. Yes, here is how to recognize the DG2 https://github.com/openvinotoolkit/openvino/blob/9e41208791aecbf12a000f5b739683631e246551/src/plugins/intel_gpu/src/plugin/plugin.cpp#L661